source/Music.cpp

```diff
@@ -254,7 +254,7 @@ void Music::Decode()
 					break;
 
 				// We'll alternate what channel we read from each time through the loop.
-				int channel = 0;
+				bool channel = false;
```

Comment:
This one I'm not a fan of, though both the before and after rely on the implicit conversion between integral and boolean types.

Reply:
Yeah same, but it's part of a group of linting warnings that I'd rather not disable.

Reply:
Would it make more sense to address the offending `channel = !channel;` statement then?